.eslintrc.js

```diff
@@ -21,7 +21,19 @@ module.exports = {
       files: ['*.test.ts', '*.test.js'],
       extends: ['@metamask/eslint-config-jest'],
     },
+    {
+      files: 'webpack.config.js',
+      rules: {
+        'node/no-unpublished-require': 0,
```

Comment:
This shouldn't be required :thinking: I'm guessing it complained about you having dev dependencies in the webpack config? It's because it doesn't know what the published files are.

Set `files` in `package.json` to just the files that get published, and this rule should be smart enough to stop complaining about dev stuff.

Reply:
yea i had it in `files`, but seemed a bit weird workaround since this repo doesn't actually publish anything to `npm`, when what I really want is to disable the rule for this particular file.

Reply:
Yeah, that's basically what setting `files` does though. In other repos where we don't publish anything (e.g. the extension), `files` is used solely for this purpose - to distinguish "dev stuff" from "prod stuff".

I guess disabling the rule works just as well though :shrug: